src/util/BitField.js

```diff
@@ -147,8 +147,7 @@ class BitField {
     if (bit instanceof BitField) return bit.bitfield;
     if (Array.isArray(bit)) return bit.map(p => this.resolve(p)).reduce((prev, p) => prev | p, 0);
     if (typeof bit === 'string' && typeof this.FLAGS[bit] !== 'undefined') return this.FLAGS[bit];
-    const error = new RangeError('BITFIELD_INVALID');
-    error.bit = bit;
+    const error = new RangeError('BITFIELD_INVALID', bit);
     throw error;
```

Comment:
```suggestion
    throw new RangeError('BITFIELD_INVALID', bit);
```
No need to do that anymore